### PR TITLE
Fix card link area

### PR DESCRIPTION
**Pull request by FieryCod**

### CODE REVIEW

File: jsconfig.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+	"compilerOptions": {
+		"target": "ES6",
+    "allowJs": true,
+     "allowSyntheticDefaultImports": true
```
Comment (k911) [inline jsconfig.json]: Please fix indentation / format document

File: jsconfig.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+	"compilerOptions": {
+		"target": "ES6",
+    "allowJs": true,
```
Comment (k911) [inline jsconfig.json]: VS Code shows 'property not allowed'

Comment (FieryCod) [reply jsconfig.json]: Thanks for check I will fix it

**Comment by FieryCod** [comment]
@zrebcu411 Could you please take a vote which is better. Maybe there is another solution and we should highlight the title or sth?

**Comment by FieryCod** [comment]
@k911 I reverted changes with underline. Should work now :)

**Pull request merged**